zoop_wrapper/wrapper/seller.py

```diff
@@ -0,0 +1,168 @@
+from ..wrapper.base import BaseZoopWrapper
+from ..models.seller import Seller
+
+
+class SellerWrapper(BaseZoopWrapper):
+    """
+    Possui os métodos do resource :class:`.Seller`
+    """
+
+    def list_sellers(self):
+        """
+        list sellers
+
+        Returns:
+            response with instances of Seller
+        """
+        url = self._construct_url(action="sellers")
+        return self._get(url)
+
+    def retrieve_seller(self, identifier):
+        """
+        retrieve seller
+
+        Args:
+            identifier: uuid id
+
+        Returns:
+            response with instance of Seller
+        """
+        url = self._construct_url(action="sellers", identifier=identifier)
+        return self._get(url)
+
+    def list_seller_bank_accounts(self, identifier):
+        """
+        list all bank accounts of some seller
+
+        Args:
+            identifier: uuid id
+
+        Returns:
+            response with instances of BankAccount
+        """
+        url = self._construct_url(
+            action="sellers", identifier=identifier, subaction="bank_accounts"
+        )
+        return self._get(url)
+
+    def _search_seller(self, id_type, identifier):
+        """
+        search seller
+
+        Args:
+            id_type: 'taxpayer_id' or 'ein'
+            identifier: value of identifier
+
+        Returns:
+            response with instance of Seller
+        """
+        url = self._construct_url(action="sellers", search=f"{id_type}={identifier}")
+        return self._get(url)
+
+    def search_business_seller(self, identifier):
+        """
+        search seller by CNPJ
+
+        Args:
+            identifier: ein (Employer Identification Number) is equivalent to CNPJ  # noqa:
+
+        Returns:
+            response with instance of Seller
+        """
+        return self._search_seller("ein", identifier)
+
+    def search_individual_seller(self, identifier):
+        """
+        search seller by CPF
+
+        Args:
+            identifier: taxpayer_id is equivalent to CPF  # noqa:
+
+        Returns:
+            response with instance of Seller
+        """
+        return self._search_seller("taxpayer_id", identifier)
```

Comment:
O que acontece se for passado errado algo aqui, especialmente se for algo trocado, digo onde era para passar um CNPJ for passado um CPF válido? O pycpfcnpj vai detectar, ou algo irá detectar? Ou só lá na api da Zoop que vai dar o erro? Posso esta falando besteira, ou isso pode vir a ser um bug bem chato de achar caso ocorra.

Reply:
Bem, ele vai bater na api da zoop e a zoop vai dizer que ou é inválido ou que não existe para esse `identifier`. 

Talvez seja uma boa validar antes de fazer o request

Reply:
Certo, você é que esta por dentro.